pkg/util/subnet/cluster_subnets.go

```diff
@@ -44,26 +44,27 @@ func (m *kubeManager) List(ctx context.Context) ([]Subnet, error) {
 	subnetMap := []Subnet{}
 
 	// select all workers by the  machine.openshift.io/cluster-api-machine-role: not equal to master Label
+	// TODO: is this even necessary? Masters should not have a MachineSet
```

Comment:
...yet. I think this is eventually coming though?

Reply:
As far as I know Masters doe have a machineset. https://docs.openshift.com/container-platform/4.12/machine_management/control_plane_machine_management/cpmso-about.html

Reply:
verified on shared cluster (4.13) that this is the case, though I don't know if that is representative of a fully "active" control plane machine management example, as the CPMS is Inactive.